front/projecttask.form.php

```diff
@@ -66,6 +66,21 @@
         //TRANS: %s is the user login
         sprintf(__('%s adds a task'), $_SESSION["glpiname"])
     );
+
+    if (isset($_POST['teammember_list'])) {
+        $taskteam = new ProjectTaskTeam();
+        foreach ($_POST['teammember_list'] as $teammember) {
+            $teammember_info = preg_split("/[_-]/", $teammember);
+            $itemtype = substr(ucfirst($teammember_info[0]), 0, -1);
+            $taskteam->add(
+                [
+                    'projecttasks_id' => $task_id,
+                    'itemtype'        => $itemtype,
+                    'items_id'        => $teammember_info[2]
+                ]
+            );
+        }
+    }
```

Comment:
This should be done in the `ProjectTask::post_addItem()` method.

Reply:
Also, I think it is already done by the `AbstractRightDropdown::getPostedIds()` methods.
Can you try to use it @Lainow ?

Also FYI @cedric-anne the "AbstractRightsDropdown" seems to be the easiest way to generate a multi item dropdown without rewriting a lot of code.
(In the future, not in this PR) We should look to rename it to something more generic (`MultiItemtypeDropdown` ?) and add a few improvements (for example we could probably use a single generic front file, making new uses of this class ever easier)